src/components/DuosFooter.js

```diff
@@ -13,6 +14,7 @@ class DuosFooter extends Component {
             li({ className: "footer-links__item" }, [a({ target: '_blank', href: "https://www.broadinstitute.org/privacy-policy" }, ["Privacy Policy"]),]),
             li({ className: "footer-links__item" }, [a({ target: '_blank', href: "https://www.broadinstitute.org/terms-conditions" }, ["Terms of Service"]),]),
             li({ className: "footer-links__item" }, [a({ href: supportLink }, ["Support"]),]),
+            li({ className: 'footer-links__item' }, [a({ href: 'status' }, ['Status'])])
```

Comment:
I would have liked to make this a `Link`, but the footer is outside the routes structure. Actually, I think this might be a bug ... `status` will break on deep page links. Will make this root relative.